examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/reid/__init__.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import absolute_import
```

Comment:
Please add copyright to **every** python code file.

Reply:
If the code is developed by you, the copyright should be from "KubeEdge", you can refer to a ianvs code file, like [this](https://github.com/kubeedge/ianvs/blob/main/core/testenvmanager/testenv/testenv.py)

Reply:
Thanks for your suggestion, I have added the copyright to every code file.